src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
+  border: $card-border;
+  border-radius: 5px;
+  box-sizing: border-box;
+  cursor: pointer;
+  transition-duration: $transition-duration;
```

Comment:
The variable `$transition-duration` is not defined in this file. Ensure it is defined in a global SCSS file or imported correctly.